build/build-bundle.js

```diff
@@ -92,6 +98,13 @@ async function browserifyFile(entryPath, distPath) {
     bundle = bundle.require(gatherer, {expose: gatherer.replace(driverPath, '../gather/')});
   });
 
+  // HACK: manually include the lighthouse-plugin-publisher-ads audits.
+  if (isDevtools(entryPath)) {
```

Comment:
kinda feels like we should resolve those functions to booleans called `isDevtools` and call the functions `isDevtoolsEntry` or something

Reply:
> kinda feels like we should resolve those functions to booleans called `isDevtools` and call the functions `isDevtoolsEntry` or something

yeah, I had a similar thought. Not sure why they're like that...I assume it made sense as a one-off and we've just c/p it multiple times now